### ISSUE TITLE
Cannot Find Option to Save/Restore settings

### ISSUE THREAD

**Issue by TheSQLGuru** [winsiderss/systeminformer#1974]
### Description of the feature, modification, idea or suggestion

Self-explanatory: the application has a LOT of settings options. There should be simple way in the app to both back them up and also restore them.

**Comment by Vasilich** [comment]
Is copy of settings.xml and usernotesdb.xml from %appdata%\SystemInformer ( or in case of portable installation SystemInformer.exe.settings.xml and usernotesdb.xml from installed folder) aren't simple enough for you?

**Comment by creizlein** [comment]
Just copy the configuration file. Is as portable as it gets.

Or you are looking for a File / Save as option alike to save the config somewhere else?

Everything is contained within a single xml. Seems really convenient

**Comment by eugenesvk** [comment]
> Or you are looking for a File / Save as option alike to save the config somewhere else?

Yes, this way you don't need to remember that there are 2 files, only remember that there is a command to export/import
